### PR TITLE
fozzie-components@v7.27.0 - GitHub Actions Browser Tests

### PR DESCRIPTION
**Pull request by siggerzz**
This PR adds WebDriverIO testing in GitHub Actions when running against the `master` branch.

Note: Whilst I have tested Visual Regression, I've removed these for now as it'll cause a clash when screenshots are uploaded. We'll look to add these once we've completely migrated away from CircleCI.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +18,7 @@ v7.26.0
 - Codeowners updated to point to new Core Team for mono-repo config, atoms and core owned components (like `f-header`/`f-footer`).
   Test files are set to be owned by the QE Reviewers team.
 
+>>>>>>> 40e028127c1433d2457a010ce84ff32ffd7087ca
```
Comment (jamieomaguire) [inline CHANGELOG.md]: might need to delete this line!

**Review by jamieomaguire — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ashleynolan) [reply CHANGELOG.md]: Looks good once this has been removed

Comment (siggerzz) [reply CHANGELOG.md]: fixed!

**Review by FayeCarter — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -345,7 +345,8 @@ jobs:
     steps:
       - attach_workspace:
           at: .
-      - browser-tools/install-chrome
+      - browser-tools/install-chrome:
+          chrome-version: 104.0.5112.101
```
Comment (jamieomaguire) [inline .circleci/config.yml:349]: are we going to have to regularly update this version?

Comment (siggerzz) [reply .circleci/config.yml:349]: Yep! Unfortunately GH Actions only supports 104 at the moment, but as soon as they have a proper mechanism to specify the Chrome version, we can switch this to an env var so it's consistent across both CI's.

Comment (jamieomaguire) [reply .circleci/config.yml:349]: makes sense - cheers!

**Review by jamieomaguire — APPROVED** [review]

**Pull request merged**